Error in "exo compute nlb show"

This issue is related to #464.

Now that #464 is fixed, the snippet

```sh
echo "Waiting for load-balancer..."
until [[ $(exo compute nlb list | wc -l) -gt 0 ]]; do
    sleep 1s
done
_nlb_id="$(exo compute nlb list --output-template '{{.ID}}')"
echo "Load-balancer ID: ${_nlb_id}"
_nlb_ip=$(exo compute nlb show "$_nlb_id" --output-format json | jq -r '.ip_address')
echo "Load-balancer IP: ${_nlb_ip}"
```
results in

```
Waiting for load-balancer...
Load-balancer ID: 2c8e3001-e403-40f7-a87e-90cb5826b0aa
panic: runtime error: invalid memory address or nil pointer dereference
[signal SIGSEGV: segmentation violation code=0x1 addr=0x0 pc=0xfdb491]

goroutine 1 [running]:
github.com/exoscale/cli/cmd.(*nlbShowCmd).cmdRun(0xc00051eb10, 0xc00091fce8?, {0xc00091fcc0?, 0x0?, 0x0?})
	github.com/exoscale/cli/cmd/nlb_show.go:129 +0x391
github.com/spf13/cobra.(*Command).execute(0xc00093cf00, {0xc000740330, 0x3, 0x3})
	github.com/spf13/cobra@v1.1.3/command.go:852 +0x67c
github.com/spf13/cobra.(*Command).ExecuteC(0x1f37ec0)
	github.com/spf13/cobra@v1.1.3/command.go:960 +0x39c
github.com/spf13/cobra.(*Command).Execute(...)
	github.com/spf13/cobra@v1.1.3/command.go:897
github.com/exoscale/cli/cmd.Execute({0x1802dfc?, 0x0?}, {0x18034c8?, 0xc0000021a0?})
	github.com/exoscale/cli/cmd/root.go:101 +0x218
main.main()
	github.com/exoscale/cli/main.go:34 +0x46
Load-balancer IP: 
```
I guess that `nlb list` returns a result before an IP got assigned and that waiting for `nlb show` will produce the error above until the IP is assigned.

--- a/CHANGELOG.md
+++ b/CHANGELOG.md
@@ -1,5 +1,12 @@
 # Changelog
 
+## 1.59.2
+
+### Bug Fixes
+
+- Fix panic in nlb show if a NLB doesn't have an IP yet (#473)
+- Remove SOS certs that were shipped as a workaround with Windows releases (#470)
+
 ## 1.59.1
 
 ### Bug Fixes

--- a/cmd/nlb_show.go
+++ b/cmd/nlb_show.go
@@ -126,7 +126,7 @@ func (c *nlbShowCmd) cmdRun(_ *cobra.Command, _ []string) error {
 		Description:  defaultString(nlb.Description, ""),
 		CreationDate: nlb.CreatedAt.String(),
 		Zone:         c.Zone,
-		IPAddress:    nlb.IPAddress.String(),
+		IPAddress:    defaultIP(nlb.IPAddress, ""),
 		State:        *nlb.State,
 		Services:     svcOut,
 		Labels: func() (v map[string]string) {

